tests/app/test_utils.py

```diff
@@ -76,3 +78,17 @@ def test_midnight_n_days_ago(current_time, arg, expected_datetime):
 
 def test_format_sequential_number():
     assert format_sequential_number(123) == "0000007b"
+
+
+def create_failed_login_for_test(notify_db_session, ip):
+    failed_login = FailedLoginCountByIP(ip=ip, attempted_at=datetime.now())
+    notify_db_session.add(failed_login)
+    notify_db_session.commit()
+
+
+@pytest.mark.parametrize(
+    "ip",
+    ["127.0.0.1", "192.0.2.15", "192.0.2.30"],
+)
+def test_check_ip_should_be_throttled(ip):
+    assert check_ip_should_be_throttled(ip) is True
```

Comment:
Could we have a test for the inverse of this to make sure we're not seeing "True" all the time?